code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -610,6 +631,36 @@
 		ai_alerts()
 	return 1
 
+/mob/living/silicon/ai/proc/borgmanagement() //Shamelessly Stolen from Botcall()
+	set category = "AI Commands"
+	set name = "Access Cyborg Management"
+	set desc = "Provide Pathing, Remote Communiques, and other data on linked synthetics."
+	if(incapacitated())
+		return
+	if(control_disabled)
+		to_chat(src, "<span class='warning'>Wireless control is disabled.</span>")
+		return
+	var/turf/ai_current_turf = get_turf(src)
+	var/ai_Zlevel = ai_current_turf.get_virtual_z_level()
+	var/c
+	c += "<A HREF=?src=[REF(src)];cyborgrefresh=1>Query network status</A><br>"
+	c += "<table width='100%'><tr><td width='40%'><h3>Name</h3></td><td width='30%'><h3>Status</h3></td><td width='30%'><h3>Location</h3></td><td width='10%'><h3>Control</h3></td></tr>"
+
+	for(var/mob/living/silicon/robot/G in connected_robots)
+		if(G.get_virtual_z_level() == ai_Zlevel && !G.emagged) //Only non-emagged Cyborgs on the same Z-level are detected!
+			c += "<tr><td width='30%'>Designation: [G.name]</A> (Module: [G.designation])</td>"
+			c += "<td width='30%'>Integrity: [G.health]% 	 		Flagged as: [G.busystatus ? "Occupied" : "Unoccupied"]		Current Objective: [objectiveupdate]</td>"
```

Comment:
```suggestion
			c += "<td width='30%'>Integrity: [G.health]% 	 		
			<br>Flagged as: [G.busystatus ? "Occupied" : "Unoccupied"]		
			<br>Current Objective: [objectiveupdate]</td>"
```

Reply:
Suggestion gives off errors for unterminated text, and missing left-hand argument. I agree the status section of the UI is yucky. I'm open to alternative suggestions.

Reply:
Ah, fucked up and didn't make the comment multiline

Reply:
That does the trick. It is a UI, so, it reads fine in the initial size, and if they want it drawn out, they can draw it out.

Reply:
Sounds good!